hs_core/templates/pages/collaborate.html

```diff
@@ -339,40 +339,18 @@ <h4 class="modal-title" id="myModalLabel">Create New Group</h4>
 
             $('.group-thumbnails').find('div.group-container').height("initial");   // Reset height
 
-            var delta;
-            if (current == "xs") {
-                return; // Delta is 1. No need to adjust
-            }
-            else if (current == "sm") {
-                delta = 2;
-            }
-            else if (current == "md" || current == "lg") {
-                delta = 3;
-            }
-
             var containers = $('.group-thumbnails').find('div.group-container');
 
+            var maxHeight = 0;
             // Fix height of containers for each row. One row every delta items.
-            for (var i = 0; i < containers.length - delta; i += delta) {
```

Comment:
@mauriel133 The comment above, particularly on ```one row every delta items``` does not apply any more due to your changes.